docs/1.getting-started/5.seo-meta.md

```diff
@@ -6,7 +6,7 @@ navigation.icon: i-ph-file-search-duotone
 
 ## Defaults
 
-Out-of-the-box, Nuxt provides sane defaults, which you can override if needed.
+Out-of-the-box, Nuxt provides same defaults, which you can override if needed.
```

Comment:
I think this is actually correct as is, with 'sane' meaning something like ' sensible' but we can update to avoid confusion.

```suggestion
Out-of-the-box, Nuxt provides sensible defaults, which you can override if needed.
```